test/integration_test.js

```diff
@@ -1422,10 +1569,119 @@ describe("Integration tests", function() {
                 }))
               )
               .should.become([
-                // For SERVER_WINS strategy, local version is marked as synced
-                { title: "task1-remote", _status: "synced" },
+                // For CLIENT_WINS strategy, local version is marked as synced
+                { title: "task1-local", _status: "synced" },
               ]);
           });
+
+          it("should put the remote database in the expected state", () => {
+            return getRemoteList(tasksTransformed._name).should.become([
+              // local task4 should have been published to the server.
+              { title: "task1-local!", done: false },
+            ]);
+          });
+        });
+
+        describe("SERVER_WINS strategy", () => {
+          beforeEach(() => {
+            return tasks
+              .sync({ strategy: Kinto.syncStrategy.SERVER_WINS })
+              .then(res => {
+                syncResult = res;
+              });
+          });
+
+          it("should have an ok status", () => {
+            expect(syncResult.ok).eql(true);
+          });
+
+          it("should contain no errors", () => {
+            expect(syncResult.errors).to.have.length.of(0);
+          });
+
+          it("should have a valid lastModified value", () => {
+            expect(syncResult.lastModified).to.be.a("number");
+          });
+
+          it("should have the outgoing conflict listed", () => {
+            expect(syncResult.conflicts).to.have.length.of(0);
+          });
+
+          it("should not skip records", () => {
+            expect(syncResult.skipped).to.have.length.of(0);
+          });
+
+          it("should not import anything", () => {
+            expect(syncResult.created).to.have.length.of(0);
+          });
+
+          it("should not publish anything", () => {
+            expect(syncResult.published).to.have.length.of(0);
+          });
+
+          it("should not update anything", () => {
+            expect(syncResult.updated).to.have.length.of(0);
+          });
+
+          it("should list resolved records", () => {
+            expect(syncResult.resolved).to.have.length.of(1);
+            // FIXME: This should maybe resolve as `null` rather than
+            // `{id, status}`??
+            const accepted = syncResult.resolved[0].accepted;
+            expect(accepted).property("_status", "deleted");
+            expect(accepted).property("id");
```

Comment:
I wasn't sure whether to maybe extract `id` and `_status` and put them in the resolution rather than in `accepted`/`rejected`. I think I probably should add them to the resolution, since it's useful to not have to guess which one is null. I might also remove them from the accepted/rejected because I already have to make a major version, may as well go all the way.